cdc/http_status.go

```diff
@@ -160,6 +169,21 @@ func writeError(w http.ResponseWriter, statusCode int, err error) {
 	}
 }
 
+func writeErrorJSON(w http.ResponseWriter, statusCode int, err error) {
+	httpErr := httpError{Error: err.Error()}
+	js, err := json.MarshalIndent(httpErr, "", " ")
```

Comment:
```suggestion
	jsonStr, err := json.MarshalIndent(httpErr, "", " ")
```
`js` is a confusing name